libbrowser/src/libbrowser_cef.cpp

```diff
@@ -1697,6 +1697,39 @@ bool MCCefBrowserBase::GetImage(void*& r_data, int& r_length)
 	return false;
 }
 
+bool MCCefBrowserBase::GetIsSecure(void)
+{
+	CefRefPtr<CefBrowser> t_browser = GetCefBrowser();
+	if (t_browser == nil)
+		return false;
+
+	CefRefPtr<CefBrowserHost> t_host = t_browser->GetHost();
+	if (t_host == nil)
+		return false;
+
+	CefRefPtr<CefNavigationEntry> t_navigation_entry = t_host->GetVisibleNavigationEntry();
+	if (t_navigation_entry == nil)
+		return false;
+
+	CefRefPtr<CefSSLStatus> t_ssl_status = t_navigation_entry->GetSSLStatus();
```

Comment:
@runrevmark Looks like I was using `t_host` instead of `t_navigation_entry` here. Hopefully that fixes things.

Reply:
Ah great! Thanks